pueue-*: add page (#5285)
tldr-pages/tldr: pages/common/pueue-pause.md

# pueue pause

> Pause running tasks or groups.
> See also: pueue start --help.

- Pause all tasks in the default group:

`pueue pause`

- Pause a running task:

> `pueue pause {{task ID}}`

Reviewer: ```suggestion
`pueue pause {{task_id}}`
```

- Pause a running task and stop all its direct [c]hildren:

> `pueue pause -c {{task ID}}`

Reviewer: ```suggestion
`pueue pause --children {{task_id}}`
```

> - Pause all tasks in a [g]roup and prevent it from starting new tasks:

Reviewer: ```suggestion
- Pause all tasks in a group and prevent it from starting new tasks:
```

> `pueue pause -g {{group name}}`

Reviewer: ```suggestion
`pueue pause --group {{group_name}}`
```

> - Pause [a]ll tasks and prevent all groups from starting new tasks:

Reviewer: ```suggestion
- Pause all tasks and prevent all groups from starting new tasks:
```

> `pueue pause -a`

Reviewer: ```suggestion
`pueue pause --all`
```

# pueue pause

> Pause running tasks or groups.
> See also: `pueue start`.
> More information: <https://github.com/Nukesor/pueue>.

- Pause all tasks in the default group:

`pueue pause`

- Pause a running task:

`pueue pause {{task_id}}`

> - Pause a running task and stop all its direct [c]hildren:

Reviewer: ```suggestion
- Pause a running task and stop all its direct children:
```
